src/main/java/com/facebook/presto/pravega/PravegaStreamDescription.java

```diff
@@ -57,6 +57,16 @@ public PravegaStreamDescription(
         this.event = requireNonNull(event, "message is null");
     }
 
+    public PravegaStreamDescription(PravegaStreamDescription streamDescription, List<PravegaStreamFieldGroup> event)
```

Comment:
Is this part of a refactoring to make the code testable? Otherwise I'm not sure why we are making such a change in a PR about tests. There are actually a number of changes to main code, could you clarify why we are bringing them here?

Reply:
this new constructor?  yes, is needed for test code. and it could be (will be) used from 'main' code but didn't make those changes now as didn't want to refactor that now.

re: other changes, as noted above in PR description test code uses 1 or 2 of the methods
(and those in turn use many of the other methods).  seemed to be a more natural fit for these methods to be in separate class (imho).